src/config/config.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import BaseConfig from '../../config/config.json';
+import SecretConfig from '../../config/config.secret.json';
+
+const Config = Object.assign({}, BaseConfig, SecretConfig);
```

Comment:
We could use a spread operator here.  `const Config = {...BaseConfig, ...SecretConfig}`